testing/tests/filesystem.lua

```diff
@@ -393,7 +403,7 @@ love.test.filesystem.mountCommonPath = function(test)
   local mount3 = love.filesystem.mountCommonPath('userhome', 'userhome', 'readwrite')
   local mount4 = love.filesystem.mountCommonPath('userappdata', 'userappdata', 'readwrite')
   -- userdesktop isnt valid on linux
-  if love.system.getOS() ~= 'Linux' then
+  if love._os ~= 'Linux' and love._os ~= 'Android' then

```

Comment:
I think these can still use `love.system.getOS()`

Reply:
Wasn't sure about that due to this notice in tests main.lua
https://github.com/love2d/love/blob/a3c84ba409405a8648bffc5381f2aad5e63139ba/testing/main.lua#L26

Reply:
I think we can consider love.system less of an optional module (especially for the test app), more like love.filesystem, so it should be fine to keep using it.

Reply:
That being said, maybe having an `isOS(...)` function in the test framework itself could be handy so you don't have to write down multiple string comparisons in different places in the code.